modules/auxiliary/admin/mssql/mssql_enum.rb

```diff
@@ -19,12 +19,13 @@ module to work, valid administrative user credentials must be
       'Author'         => [ 'Carlos Perez <carlos_perez[at]darkoperator.com>' ],
       'License'        => MSF_LICENSE
     ))
+
   end
 
   def run
     print_status("Running MS SQL Server Enumeration...")
 
-    if !mssql_login_datastore
+    if !mssql_login(datastore['USERNAME'], datastore['PASSWORD'])
```

Comment:
You can keep `mssql_login_datastore` around in the metasploit mixin if you want - it doesn't seem too bad, as it allows us to read other datastore options that get added in the future and is backwards compatible